unit-tests/func/hw-errors/l500/l500-error-common.h

```diff
@@ -0,0 +1,85 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#include "l500/l500-private.h"
+using namespace rs2;
+
+const uint8_t l500_trigger_error_opcode = 0x5E;
+
+std::map< uint8_t, std::pair< std::string, rs2_log_severity > > build_log_errors_map()
+{
+    using namespace librealsense::ivcam2;
+
+    std::map< uint8_t, std::pair< std::string, rs2_log_severity > > res;
+    l500_notifications_types last_index = success;
+    for (auto & i : l500_fw_error_report)
+    {
+        if (i.first == success)
+            continue;  // LRS ignore if he get success as error
+
+        if (i.first > last_index + 1)
+        {
+            for (auto j = last_index + 1; j < i.first; j++)
+            {
+                std::stringstream s;
+                s << "L500 HW report - unresolved type " << j;
```

Comment:
Can we use `l500_notification_decoder` here, to avoid this knowledge of how the error is generated?

Reply:
you mean to use l500_notification_decoder::decode?
I think that if we use this code in unit test we miss the point of the unit test that meant to check this code among the other things.